helm-charts/interoperator/templates/broker.yaml

```diff
@@ -18,7 +18,7 @@ spec:
   selector:
     app: {{ .Release.Name }}-broker
 ---
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
Hi @Pooja-08, please check if this apiversion is supported in k8s versions 1.18 and 1.19 also...